Bergado_Shamma/index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style2.css" rel="stylesheet" />
+    <title>Bergado Shamma Page</title>
+  </head>
+  <body>
+    <div class="name">
+      <img title="This is my name" src="./pictures/name.png" />
+    </div>
+
+    <div class="hobbies-and-interests">
+      <h2>My interests and hobbies</h2>
+      <ul>
+        <li class="reading">
+          <a
+            href="https://reedsy.com/discovery/blog/best-books-to-read-in-a-lifetime">
+            Reading
+          </a>
+          <p id="p">
+            "Books are a uniquely portable magic." - Stephen King
+          </p>
+        </li>
+        <li class="watching">
+          <a href="https://www.imdb.com/list/ls566830858/">Watching</a>
+          <p id="p">
+            "The best way to learn anything is by watching." - Jeff Rich
+          </p>
+        </li>
+        <li class="writing">
+          <a href="https://www.pinterest.com/nancyycrespo/calligraphynotes/">
+            Writing
+          </a>
+          <p id="p">
+            "Write what should not be forgotten." - Isabel Allende
+          </p>
+        </li>
+        <li class="painting">
+          <a href="https://www.pinterest.com/bayoubrushes/painting-ideas/">
+            Painting
+          </a>
+          <p id="p">
+            "Every artist was first an amateur." - Ralph Waldo Emerson
+          </p>
+        </li>
+        <li class="singing">
+          <a href="https://www.karafun.com/karaoke/styles/popmusic/">
+            Singing
+          </a>
+          <p id="p">
+            "Songs are like tattoos—you know, they're with you forever." 
+            - Bob Dylan
```

Comment:
These id "p" seems not to be in used in either of your css or js files. If you do not need these, you can safely remove the you can safely remove the id="p" attribute from those p elements to clean up your code.